tasks/setup-Ubuntu.yml

```diff
@@ -4,17 +4,28 @@
     name: dirmngr
     state: present
 
-- name: Add PPA for Nginx (if configured).
+- name: Remove legacy ppa if present
   apt_repository:
-    repo: 'ppa:nginx/{{ nginx_ppa_version }}'
-    state: present
-    update_cache: true
-  register: nginx_ppa_added
-  when: nginx_ppa_use | bool
+    repo: ppa:nginx/stable
+    state: absent
+
+- name: Add nginx public key
+  apt_key: url=https://nginx.org/keys/nginx_signing.key
+           state=present
+  register: nginx_repo_added
+  when: nginx_repo_use | bool
+
+- name: Add nginx stable repository
+  apt_repository: repo='deb http://nginx.org/packages/{{ '/mainline' if nginx_repo_mainline else ''}}ubuntu {{ ansible_distribution_release }} nginx'
```

Comment:
Should `'/mainline'` be `'mainline/'` instead?  Just double-checking that the slash is in the correct spot here

Reply:
you are right, thanks for the suggestion 👍